### PR TITLE
Rename `Autofix` to `Fix`

### PR DESCRIPTION
**Pull request by konstin**
**Summary** Mostly mechanical symbol rename and search-and-replace, with small changes to the markdown docs to read better

### CODE REVIEW

**Comment by konstin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #7657** <a href="https://app.graphite.dev/github/pr/astral-sh/ruff/7657" target="_blank">Graphite</a>  👈
    * **PR #7658** <a href="https://app.graphite.dev/github/pr/astral-sh/ruff/7658" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/astral-sh/ruff/7657?utm_source=stack-comment).

**Review by charliermarsh — APPROVED** [review]
I'm comfortable with this, but probably worth getting at least one more approval before moving forward. I would also be interested in Zanie's opinion as the point person on CLI and applicability and such.

File: .github/release.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ changelog:
     - title: Rules
       labels:
         - rule
-        - autofix
+        - fix
```
[the remark this answers is not in the document]
Comment (zanieb) [reply .github/release.yml:15]: This is a little tricky as it seems like `fix` means the change is a bug fix. Do we use the `autofix` label in the changelog much anyway?

**Comment by zanieb** [comment]
I'll try to review this tomorrow morning!

**Comment by MichaReiser** [comment]
Makes sense to me, considering that we'll have "Autofix" with applicability "Manual"

**Review by zanieb — APPROVED** [review]

**Comment by konstin** [comment]
What's the decision on what to do with the labels, should we keep them as-is?

**Comment by zanieb** [comment]
I don't think we need to generate a section for the `fix` label — seems safe to remove.

**Comment by charliermarsh** [comment]
Yeah, I think it's fine to remove from the release notes. Do we want to remove the label entirely though? I'm fine with that too.

**Comment by zanieb** [comment]
> Yeah, I think it's fine to remove from the release notes. Do we want to remove the label entirely though? I'm fine with that too.

It kind of makes sense in the "Issue" section so seems fine to rename to `fix` and keep for now? We can see if there's any practical confusion

**Pull request merged**